gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/kafka/client/GobblinKafkaConsumerClient.java

```diff
@@ -166,6 +167,8 @@ default long committed(KafkaPartition partition) {
     return -1L;
   }
 
+  default void assignAndSeek(List<KafkaPartition> topicPartitions, Map<KafkaPartition, LongWatermark> topicWatermarksMap) { return; }
```

Comment:
is this relevant ?

Reply:
Yes it is. This method is used inside LI's internal Kafka consumer client.